lib/services/errors/context-absent.ts

```diff
@@ -0,0 +1,2 @@
+import TypedError = require('typed-error');
```

Comment:
I'm against this on the principal that it's a very specific error class, and we'd end up with them dotted all over the place. I'm far more in favour of including any error type that's associated with a class as a definition in the class itself (look at https://github.com/resin-io-modules/resin-procbots/blob/master/lib/services/github.ts#L60 as an example).
For that reason, I think there should be a `ProcBotError` type, that exists in `procbot.ts`. This would include both `code` and `message`, where code is from a const enumerated type defined in `procbot-types.ds`. This is therefore one such `ProcBotError` error type.